### DISCUSSION TITLE
How can I access Security Devices settings?

### DISCUSSION THREAD

**Discussion by arjan-s** [qutebrowser/qutebrowser#5887] [Q&A] [answered]
I'm having trouble with my USB token which I use for other applications than my browser. It causes slowdowns and timeouts on websites because Qutebrowser seems to be trying to use my token to authenticate me to every website. That's what it looks like at least, since for every website my token LED starts flashing like crazy, even if it's just a regular public website.
I would like to disable the token as a Security Device in Qutebrowser, like I did in Firefox.
How do I access/modify these kind of settings in Qutebrowser?

**Comment by The-Compiler** [comment, marked as answer]
I don't think anything like that is available from QtWebEngine (which handles those keys without qutebrowser doing anything about them directly).

How would you do the same thing in Chromium? I can see there is chrome://settings/securityKeys but I don't see a way to disable a key there.

**Reply by arjan-s** [reply]
Thanks for thinking along! For Chromium it's set in `$HOME/.pki/nssdb/pkcs11.txt`. However, when I remove the libeToken.so reference there, it doesn't change the behaviour of Qutebrowser.

**Reply by The-Compiler** [reply]
I have no idea then, I'm afraid... In general changes in `~/.pki` do affect qutebrowser as well (e.g. for custom client/server certificates), so I'm surprised that doesn't work.

**Reply by arjan-s** [reply]
Wait! That was it after all!
I also had a reference to `onepin-opensc-pkcs11.so` in that file, which turned out to be the one causing the token to go mad. Removing that fixed the slowdowns and timeouts.
